docs/goals-canvas.md

```diff
@@ -16,4 +16,4 @@ title: Goals Canvas
 | laag | SPECIAL!!!! | actuele verdeling  van alle gebruiksfuncties van verblijfsobjecten in BAG binnen een bepaald gebied | Zoek verblijfsobjecten standplaatsen en ligplaatsen in een gebied | invoeren contour | verdeling van actuele gebruiksdoelen in percentages (samen 100%) | actuele uitnutting gebruiksdoelen en beschikbare vrije ruimte binnen bestemmingsplan bepalen | [7](https://github.com/VNG-Realisatie/Haal-Centraal-BAG-bevragen/issues/7){:target="_blank"} |
 | 1    | | Panden zoeken | Zoek panden op contour in combinatie met bouwjaar binnen periode (voor of na bouwjaar) | contour in combinatie met bouwperiode | panden | panden vinden van voor of na een bepaald bouwjaar of binnen een periode  tbv energietransitie | [306](https://github.com/VNG-Realisatie/Haal-Centraal-BAG-bevragen/issues/306){:target="_blank"} |
 | 2    | | Panden zoeken | Zoek panden op contour in combinatie met status en geconstateerd | contour in combinatie met status en geconstateerd | panden | panden vinden in een bepaald gebied met gecontateerd tbv handhaving stedelijke ontwikkelingen en belastingzaken | [331](https://github.com/VNG-Realisatie/Haal-Centraal-BAG-bevragen/issues/331){:target="_blank"}, [333](https://github.com/VNG-Realisatie/Haal-Centraal-BAG-bevragen/issues/333){:target="_blank"} |
-| 3    | | Adresseerbaar object zoeken | zoeken ADO op contour in combinatie met oppervlakterange, status, gebruiksdoel, type en geconstateerd | contour in combinatie met oppervlakterange, status, gebruiksdoel, type en gecontateerd | Adresseerbaar objecten | ADO's vinden in een bepaald gebied met gecontateerd tbv handhaving stedelijke ontwikkelingen en belastingzaken | [19](https://github.com/VNG-Realisatie/Haal-Centraal-BAG-bevragen/issues/19){:target="_blank"}, [331](https://github.com/VNG-Realisatie/Haal-Centraal-BAG-bevragen/issues/331){:target="_blank"}, [332](https://github.com/VNG-Realisatie/Haal-Centraal-BAG-bevragen/issues/332){:target="_blank"}, [334](https://github.com/VNG-Realisatie/Haal-Centraal-BAG-bevragen/issues/334){:target="_blank"} |
+| 3    | | Adresseerbaar object zoeken | zoeken ADO op contour of woonplaats in combinatie met oppervlakterange, status, gebruiksdoel, type en geconstateerd | contour in combinatie met oppervlakterange, status, gebruiksdoel, type en gecontateerd | Adresseerbaar objecten | ADO's en geralteerde panden vinden in een bepaald gebied met gecontateerd tbv handhaving stedelijke ontwikkelingen en belastingzaken | [19](https://github.com/VNG-Realisatie/Haal-Centraal-BAG-bevragen/issues/19){:target="_blank"}, [331](https://github.com/VNG-Realisatie/Haal-Centraal-BAG-bevragen/issues/331){:target="_blank"}, [332](https://github.com/VNG-Realisatie/Haal-Centraal-BAG-bevragen/issues/332){:target="_blank"}, [334](https://github.com/VNG-Realisatie/Haal-Centraal-BAG-bevragen/issues/334){:target="_blank"} |
```

Comment:
waarom wordt hier "woonplaats" toegevoegd? Die staat vervolgens niet in de volgende kolom, waarin staat nog "contour in combinatie met ..." zonder woonplaats.

Moet ook de woonplaats als parameter worden toegevoegd?

Reply:
is het niet handig om zoeken op contour en zoeken op woonplaats gescheiden te houden?
Dus zoeken op woonplaats i.c.m. gebruiksdoel en type met GET /adresseerbareobjecten, en zoeken op contour i.c.m. gebruiksdoel en type met POST /adresseerbareobjecten/zoek

voordelen:
1. zoeken met woonplaats kan op de RESTful manier
2. zoeken met contour kan duidelijk gespecificeerd worden (contour is required)

Reply:
Volgens mij is de "of woonplaats" deen overblijfsel van de situatie dat we pas in Q3 met contour kunnen beginnen.  De gedachte was dat we dan een heel groot deel van de functionaliteit in Q2 konden ontwikkelen en ik Q3 de contour konden toevoegen.  Om in die situatie te voorkomen dat we resultaten van heel nederland zouden krijgen hebben we de woonplaats toegevoegd. 

Nu we toch contour nu al gaan doen is daarmee in deze endpoint de woonplaats obsolete geworden. 
Mocht deze wens alsnog ingevuld moeten worden, dan ben ik het een met jouw voorstel om woonplaats (met gebruiksdoel en tyype) als parameter toe te voegen aan GET /adresseerbareobjecten

Reply:
De woonplaats parameter is niet expliciet, maar wel impliciet onderdeel van #334. het gebruik van woonplaats past beter bij deze user story dan/ heeft voor deze user story de voorkeur boven contour.

Reply:
> waarom wordt hier "woonplaats" toegevoegd? Die staat vervolgens niet in de volgende kolom, waarin staat nog "contour in combinatie met ..." zonder woonplaats.
> 
Zie boven
> Moet ook de woonplaats als parameter worden toegevoegd?
Ja idd

Reply:
> is het niet handig om zoeken op contour en zoeken op woonplaats gescheiden te houden?
> Dus zoeken op woonplaats i.c.m. gebruiksdoel en type met GET /adresseerbareobjecten, en zoeken op contour i.c.m. gebruiksdoel en type met POST /adresseerbareobjecten/zoek
> 
> voordelen:
> 
> 1. zoeken met woonplaats kan op de RESTful manier
> 2. zoeken met contour kan duidelijk gespecificeerd worden (contour is required)

Ik zal een nieuwe regel toevoegen